src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -21,7 +21,7 @@ namespace Microsoft.IdentityModel.JsonWebTokens
     /// <summary>
     /// A <see cref="SecurityToken"/> designed for representing a JSON Web Token (JWT). 
     /// </summary>
-    public class JsonWebToken : SecurityToken
+    public class JsonWebToken : SecurityToken, IJsonWebToken
```

Comment:
Would be great to get more context on why JwtSecurityToken also doesn't implement IJsonWebToken

Reply:
It crossed my mind that it might be easier to work only with IJsonWebToken in OIDCProtocolValidator but I assumed that there must be a tradeoff there :)